server/main.py

```diff
@@ -9,11 +9,12 @@
 from google.cloud import secretmanager
 from google.cloud import pubsub_v1
 
+from hail import version as get_hail_version
 import hailtop.batch as hb
 from hailtop.config import get_deploy_config
 
 GITHUB_ORG = 'populationgenomics'
-
+HAIL_VERSION = get_hail_version()
```

Comment:
I actually meant the version of the Hail deployment, but I suppose the server version is also important!

Reply:
Blocked by: https://github.com/populationgenomics/hail/issues/34